src/domain.h

```diff
@@ -343,6 +345,37 @@ class Domain : public NLVObject<virDomainPtr, DomainCleanupHandler>
       unsigned int flags_;
   };
 
+  class GetMetadataWorker : public NLVPrimitiveReturnWorker<virDomainPtr, std::string> {
```

Comment:
I was suprised that it seemed to compile without me doing this but I guess it's better for completeness.